drivers/windows/inst/templates/task_run.bat

```diff
@@ -17,6 +17,7 @@ ECHO working directory: %CD%
 set R_LIBS_USER={{hipercow_library}}
 set HIPERCOW_NO_DRIVERS=1
 set RENV_AUTOLOADER_ENABLED=FALSE
+set HIPERCOW_CORES_VARIABLE_NAME=CCP_NUMCPUS
```

Comment:
Thinking about this some more, was there a reason we didn't think about doing:

```suggestion
set HIPERCOW_CORES=CCP_NUMCPUS
```

just as easy really? but we did not come up with this when talking

Reply:
Done